tests/security/selinux/setsebool.pm

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test "# setsebool" command with options "-P / -N" can work
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#63751, tc#1741287
+
+use base 'opensusebasetest';
+use power_action_utils "power_action";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    my $test_boolean = "fips_mode";
```

Comment:
Question from the setting, if system set test_boolean = fips_mode, does it mean the system is working really in fips mode? or it is only a flag in SELinux.

Reply:
It is a selinux boolean/flags, here only turn on/of the boolean system will not work in fips mode.

Reply:
May I treat it as it's only a flag for SELinux, and nothing related to system is working in fips or not.